docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/channels-reference-guide.md

```diff
@@ -24,35 +24,35 @@ Users can enable a workflow for [buyers and sellers](../../../orders-and-fulfill
 
-For _Price Type_, users can determine whether a price list or promotion is calculated using the gross price or the net price.
-
-For _Discounts Target_, users can determine whether applicable discounts are calculated based on the product's gross price or net price.
+For _Price Type_, users can determine whether product prices are displayed with either tax included (gross price) or tax excluded (net price).
 
 ![The Prices tab is where the tax categories and prices are set.](./channels-reference-guide/images/03.png)
 
 ### Health Checks
 
-Health checks allow users to quickly fix any issues with their store sites, such as missing widgets.
+Health checks allow users to quickly fix any common setup issues with their store sites, such as missing widgets that are required to have a functioning store.
 
 ![Health checks allow you to quickly add Commerce portlets or fix issues if you had built the site manually.](./channels-reference-guide/images/04.png)
 
 ### Payment Methods
 
-Users can configure which [payment methods](../../../store-administration/configuring-payment-methods/managing-payment-methods.md) to use for the store. In addition, users can use a [custom payment engine](../../../developer-guide/tutorials/implementing-a-new-payment-method.md).
+Users can configure which [payment methods](../../../store-administration/configuring-payment-methods/managing-payment-methods.md) to use for the channel. In addition, users can use a [custom payment engine](../../../developer-guide/tutorials/implementing-a-new-payment-method.md).
 
 ![The Payment Methods section allows you to choose which payment method to use.](./channels-reference-guide/images/05.png)
 
 ### Shipping Methods
 
 Users can configure the channel's shipment methods. By default, the [flat rate](../../../store-administration/configuring-shipping-methods/using-the-flat-rate-shipping-method.md) is enabled. Users can also enable a [variable rate](../../../store-administration/configuring-shipping-methods/using-the-variable-rate-shipping-method.md) shipping method. Subscribers can also enable [FedEx](../../../store-administration/configuring-shipping-methods/using-the-fedex-shipping-method.md).
 
+[//]: # (NOTE: by default the flat rate shipping method is not enabled and FedEx is not only available for subscribers. The above paragraph needs to be revised.)
+
 Lastly, users can use a [custom shipping engine](../../../developer-guide/tutorials/implementing-a-new-shipping-engine.md).
 
 ![The Shipment Methods section allows you to choose which shipment method to use.](./channels-reference-guide/images/06.png)
 
 ### Tax Calculations
 
-Users can configure the channel's tax calculations. Upon creation, Tax Calculations are inactive. To learn more activating them, see [Setting Tax Rate by Address](../../../store-administration/configuring-taxes/setting-tax-rate-by-address.md) or [Setting Tax Rate by Fixed Rate](../../../store-administration/configuring-taxes/setting-tax-rate-by-fixed-rate.md)
```

Comment:
Extraneous. In other settings where the setting is inactive/disabled by default, we don't call it out. Not all sellers need to calculate tax, hence why we allow the option to enable/disable tax methods.

Reply:
Revised